### PR TITLE
Fixup UDS failing tests.

### PR DESCRIPTION
**Pull request by markan**
Fix some tests that weren't being run in CI and thus missed.

### CODE REVIEW

File: agent/config/testdata/full-config.json [hunk 1]
```diff
@@ -593,6 +590,11 @@
             "destination_type": "prepared_query",
             "local_bind_address": "127.24.88.0",
             "local_bind_port": 11884,
```
Comment (rboyer) [inline agent/config/testdata/full-config.json:592]:
```suggestion
            "local_bind_port": 11884
```

**Review by dnephin — APPROVED** [review]
LGTM

**Comment by hc-github-team-consul-core** [comment]
:cherries: If backport labels were added before merging, cherry-picking will start automatically.

To retroactively trigger a backport after merging, add backport labels and re-run https://circleci.com/gh/hashicorp/consul/363350.

**Comment by hc-github-team-consul-core** [comment]
:cherries::x: Cherry pick of commit d93dbb866f7443019e5aeff735cabd7e5112953d onto `release/1.10.x` failed! [Build Log](https://circleci.com/gh/hashicorp/consul/363350)

**Pull request merged**